CONTRIBUTING.md

```diff
@@ -136,6 +135,60 @@ You can import the Elasticsearch project into IntelliJ IDEA via:
  - In the subsequent dialog navigate to the root `build.gradle` file
  - In the subsequent dialog select **Open as Project**
 
+### Importing the project into Eclipse
+
+IMPORTANT: If you have previously imported the project by running `./gradlew eclipse`
+           then you must build an entirely new workspace and `git clean -xdf` to
+           blow away *everything* that the gradle eclipse plugin made.
+
+ - Select **File > Import...**
+ - Select **Existing Gradle Project**
+ - Select **Next** then **Next** again
+ - Set the **Project root directory** to the root of your elasticsearch clone
+ - Click **Finish**
+
+This will spin for a long, long time but you'll see many errors about circular
+dependencies. Fix them:
+
+ - Select **Window > Preferences**
+ - Select **Java > Compiler > Building**
+ - Set **Circular dependencies** to **Warning**
+ - Apply that and let the build spin away for a while
```

Comment:
Had to do the Refresh/Cleanup dance once but I guess that's what everybody expects?

Reply:
I think accepting this change will prompt you to let it rebuild. I didn't do it when writing the instructions but it is what I remember. If you need to refresh or clean up we should so. I mostly don't have to do that though.